src/mpq.rs

```diff
@@ -203,17 +202,7 @@ pub struct ParseMpqError {
 
 impl fmt::Display for ParseMpqError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        self.description().fmt(f)
-    }
-}
-
-impl Error for ParseMpqError {
```

Comment:
You have removed these impls completely, which is neither desirable nor backwards-compatible. The soft-deprecation of `Error::description` means you can use an empty impl, not no impl.

Reply:
Oh cool, thanks for the info. I will fix and re-spin.